assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -545,6 +662,21 @@ const baseSelectors = {
 		// update.
 		return state.isAwaitingModulesRefresh;
 	} ),
+
+	canActivateModule( state, slug ) {
```

Comment:
Selectors like this should be documented.

Reply:
Still needs to be addressed.

Reply:
Addressed.